app/core/models.py

```diff
@@ -327,3 +328,27 @@ def save(self, *args, **kwargs):
             raise ValidationError('There can only be one '
                                   'SenderEmailConfiguration instance')
         return super(SenderEmailConfiguration, self).save(*args, **kwargs)
+
+
+class Course(models.Model):
+    """Model to store course instances for interest lists"""
```

Comment:
I'm wondering if we're being too concrete by calling this a "Course" -- we talk a lot about Learning Experiences and how a Course is just one type of experience.  I mean, I get that we have used the "course" term all over the place, but I think that's more because of the "Enterprise Course Catalog" idea versus the "Open Learning Experience Platform" idea.

Reply:
Fair enough, I'll rename this model to experience